jekyll/_cci2/vs-code-extension-overview.adoc

```diff
@@ -11,34 +11,176 @@ contentTags:
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_config_helper_on-hover-documentation.png[Screenshot showing the contextual information on hover]
+
+- **Syntax validation**. This helps you identify typos, incorrect use of parameters, incomplete definitions, wrong types, invalid or deprecated machine versions, etc.
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_config_helper_syntax-validation.gif[Screenshot showing the synthax highlightning when an error is identified]
+
+//TODO: Is the missing keys something like a key hint?
+- **Usage warnings**. This helps you identify deprecated parameters, unused jobs or executors, or missing keys that prevent you from taking advantage of CircleCI’s full capabilities.
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_config_helper_usage-warning.png[Screenshot showing code highlightning to warn on an unused job]
+
+- **Auto completion**. This is available with both built-in keys and parameters as well as user-defined variables.
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_config_helper_autocomplete.png[Screenshot showing two suggestions to autocomplete the line of code]
+
+NOTE: The config helper is based on a dedicated language server specific for CircleCI YAML files, and is open source. You can view its source code, contribute and add issues directly on the project repository: link:https://github.com/CircleCI-Public/circleci-yaml-language-server[circleci-yaml-language-server].
+
+[#config-validation-commands]
+=== Config validation commands
+
+The config helper also provides two commands that help you statically validate your YAML config files without having to run a pipeline.
+
+* Validate current configuration file
++
+Corresponds to the CLI command `circleci config validate`, and verifies statically that the config file is well formed. Please note that this command only validates this file for structure and syntax errors, but not for semantic error (e.g. "This job does not exist").
+
+* Validate current configuration file against org policy
++
+Corresponds to the CLI command `circleci policy decide`, and verifies that the configuration file complies with your organisation policies (if any are set).
+
+Both of these commands can be invoked by:
+
+- Right-clicking on a CircleCI YAML file.
+
+- Clicking on the CircleCI button on the top-right corner of a tab in which a `.circleci/config.yml` is open. Note that the button will not be visible if you are editing any other file.
+
+//TODO: Verify we need an additional section for the language server or if 
+//the banner earlier in the config helper section is sufficient
+[#open-source-language-server]
+=== Open source language server
+
+The capabilities of the VS Code extension are open to all code editors. You can fork the 
```

Comment:
Not all extension capabilities depend on the Language Server, so we need to change this. 
I also think it makes sense to keep all the Language Server related info in one place. 
For example

> NOTE: The config helper is based on a dedicated language server specific for CircleCI YAML files, and is open source. You can view its source code, contribute and add issues directly on the project repository: link:https://github.com/CircleCI-Public/circleci-yaml-language-server[circleci-yaml-language-server].
You can also integrate the Language Server into any editor which supports the Language Server Protocol, and build your own plugin to benefit to from Config Helper capabilities in your favourite editor.

Reply:
@BeFunes I'm going to remove the earlier NOTE and just have everything about the language server in this section. Thanks!